content/docs/1.10.0/snapshots-and-backups/setup-a-snapshot.md

```diff
@@ -3,12 +3,224 @@
+
+#### Step 1 – Delete or detach the original volume
+
+If you are restoring over the same name or want a clean restore:
+
+```bash
+kubectl delete pvc longhorn-test-pvc -n default
+```
+
+#### Step 2 – Create a new PVC referencing the snapshot
+
+Create `longhorn-pvc-restore.yaml`:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: restored-pvc
+  namespace: default
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: longhorn
+  dataSource:
+    name: longhorn-test-snapshot      # Snapshot CR name
+    kind: Snapshot
+    apiGroup: longhorn.io
+  resources:
+    requests:
+      storage: 2Gi
+```
+
+Apply it:
+
+```bash
+kubectl apply -f longhorn-pvc-restore.yaml
+```
+
+Then verify:
+
+```bash
+kubectl get pvc restored-pvc -n default
+```
+
+> **Note**:
+> If you manage volumes directly (not via PVCs), you can also create a new `Volume` CR with the field:
+>
+> ```yaml
+> spec:
+>   fromSnapshot: longhorn-test-snapshot
+> ```
+
+
+### 4. Deleting a Snapshot
+
+To remove the snapshot CR:
+
+```bash
+kubectl delete snapshot.longhorn.io longhorn-test-snapshot -n longhorn-system
+```
+
+Expected output:
+
+```
+snapshot.longhorn.io "longhorn-test-snapshot" deleted
+```
+
+Longhorn automatically handles the cleanup of the underlying data according to its internal retention and replica policies.
+
+### Key CR Fields & Their Meanings
+
+Below are key fields for `Snapshot` CRs (Longhorn `v1beta2`):
+
+| Field                 | Location               | Description                                                         |
+| --------------------- | ---------------------- | ------------------------------------------------------------------- |
+| `spec.volume`         | `.spec.volume`         | Name of the Longhorn volume being snapshotted.                      |
+| `spec.createSnapshot` | `.spec.createSnapshot` | Boolean — triggers snapshot creation.                               |
+| `spec.labels`         | `.spec.labels`         | Optional user-defined labels for organizing or filtering snapshots. |
+| `status.createdAt`    | `.status.createdAt`    | Timestamp when the snapshot was created.                            |
+| `status.readyToUse`   | `.status.readyToUse`   | Indicates if the snapshot is available for restore operations.      |
+| `status.restoreSize`  | `.status.restoreSize`  | Size of the restored data (bytes).                                  |
+| `status.parent`       | `.status.parent`       | The parent snapshot (if part of a snapshot tree).                   |
+| `status.children`     | `.status.children`     | Child snapshots branching from this snapshot.                       |
+| `status.removed`      | `.status.removed`      | Whether the snapshot has been marked for removal/cleanup.           |
+
+> **Tip**:
+> Inspect snapshot details directly:
+>
+> ```bash
+> kubectl get snapshot.longhorn.io longhorn-test-snapshot -n longhorn-system -o yaml
+> ```
```

Comment:
Users can get the information from the API defintion. We can remove the session.